app/src/main/res/layout/layout_profile_header.xml

```diff
@@ -10,6 +11,22 @@
   app:layout_constraintTop_toTopOf="parent"
   app:layout_constraintVertical_bias="0.0">
 
+  <Button

```

Comment:
Could you change the color and size of the button to make it a bit less eye catching for a sign out button 😄 And maybe change the text to _sign out_ or just a sign out icon to make it a bit more clear what the button does.